src/middlewared/middlewared/utils/osc/linux/app.py

```diff
@@ -16,7 +16,7 @@ def get_app_version():
 
     return {
         'stable': not RE_UNSTABLE.match(manifest['version']),
-        'version': manifest['version'][len('TrueNAS-'):],
-        'fullname': manifest['version'],
+        'version': manifest['version'],
+        'fullname': f"TrueNAS-{manifest['version']}",
```

Comment:
What system.version looks like after this?

Reply:
`MASTER-20200603-042400`, do you want `13.0-MASTER-20200603-042400` instead?

Reply:
We need some mention of SCALE.

Currently it is: midclt call system.version
TrueNAS-12.0-MASTER-202006030424

We probably want something along the lines of:
TrueNAS-SCALE-XX.XX-MASTER-202006030424